modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,263 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
 In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
 
+[id="pipelines-new-features-1-8_{context}"]
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an {product-title} cluster that is running on ARM hardware. This includes support for `ClusterTask` resources and the `tkn` CLI tool.
++
+include::snippets/technology-preview.adoc[]
+// Shubham Minglani
+
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
```

Comment:
this is Pipelines as Code feature shall we move it to Pipelines as Code section?

Reply:
Thanks. Moved it to the `=== {pac}` section and combined it with a duplicate entry. Now, this reads as follows:

* With this update, as a developer, you can use the new `tkn-pac setup-webhook cli` command to add a Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
// (link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
// Shivam Mukhade